src/Cli/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.cs.xlf

```diff
@@ -52,6 +52,11 @@
         <target state="translated">Nástroj {0} byl úspěšně aktualizován z verze {1} na verzi {2} (soubor manifestu {3}).</target>
         <note />
       </trans-unit>
+      <trans-unit id="UpdateSucceededVersionNoChange">
```

Comment:
Is this from your previous PR?

Reply:
No, this is not up to date and merged with main yet, this is how it was before that PR was merged in 👍